process-engine-plugin/failed-job-retry-profile/src/test/java/org/camunda/bpm/example/test/FailedJobRetryProfileTest.java

```diff
@@ -40,7 +40,8 @@
 
 public class FailedJobRetryProfileTest {
 
-  protected static final int DEFAULT_RETRIES = 3;
+  protected static final int MAX_CRM_RETRIES = 5;
```

Comment:
fixed unit test to use correct initial job retry values. See related [docs update PR](https://github.com/camunda/camunda-docs-manual/pull/1791/files)